debug_tool/utils/db.py

```
@@ -98,27 +109,140 @@ def get_overview(self) -> Dict[str, Any]:
+            )
         try:
             collection = self.client.get_collection(collection_name)
             return {"count": int(collection.count())}
-        except Exception:
-            return {"count": 0}
+        except Exception as e:
+            logger.warning("[调试工具] ChromaDB集合统计查询失败(%s)，降级到SQLite: %s", collection_name, e)
+            return self._get_collection_stats_via_sqlite(collection_name)
+
+    def _browse_collection_via_sqlite(self, collection_name: str, limit: int = 20, offset: int = 0):
+        """绕过 chromadb API，直读 sqlite3 浏览集合内容（版本兼容回退）"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)